From 075842d9affb4c0be636c485ad48a2ca7e258c38 Mon Sep 17 00:00:00 2001
From: hyoyun-Kim <hyo-yun1103@daum.net>
Date: Mon, 29 Nov 2021 15:33:20 +0900
Subject: [PATCH 2/2] iolibrary_driver_ftp_client

---
 Internet/FTPClient/ftpc.h | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/Internet/FTPClient/ftpc.h b/Internet/FTPClient/ftpc.h
index ef2be52..379e339 100644
--- a/Internet/FTPClient/ftpc.h
+++ b/Internet/FTPClient/ftpc.h
@@ -27,18 +27,20 @@ extern "C" {
  * Below is an example of a function of lpc_chip library. */
 //#define ftp_getc()	Board_UARTGetCharBlocking()
 
+#define ftp_getc() getchar()
+
 #ifdef F_FILESYSTEM
 #include "ff.h"
 #endif
 
-#ifndef	ftp_getc()
-#define Need_UARTGetCharBlocking_func
-#else
-/* Change library
- * Change!! -> board_api.h,
- * Below is an example of a function of lpc_chip library. */
-#include "board_api.h"
-#endif
+// #ifndef	ftp_getc()
+// #define Need_UARTGetCharBlocking_func
+// #else
+// /* Change library
+//  * Change!! -> board_api.h,
+//  * Below is an example of a function of lpc_chip library. */
+// #include "board_api.h"
+// #endif
 
 
 #define LINELEN		100
-- 
2.32.0.windows.2

